app/javascript/views/userDetail/Notes.jsx

```diff
@@ -1,20 +1,38 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { Rolodex, Card, CardBody, CardHeader, CardTitle } from '@cwds/components'
+import { Rolodex, Card, CardBody, CardHeader, CardTitle, Input } from '@cwds/components'
 
-const Notes = ({ userNotes }) => (
+const Notes = ({ isUserEditable, userNotes, notesWordCount, onChange }) => (
   <Rolodex>
     <Card>
       <CardHeader>
         <CardTitle style={{ fontSize: '1.75rem', fontWeight: 400 }}>Notes</CardTitle>
       </CardHeader>
-      <CardBody className="pt-0" />
+      <CardBody className="pt-0">
+        {isUserEditable ? (
+          <div>
+            <Input
+              type="textarea"
+              className="form-control text-area"
+              value={userNotes}
+              onChange={event => onChange('notes', event.target.value)}
+              maxLength={250}
+            />
+            <div className="pull-right">Characters used: {`${notesWordCount}/ 250`}</div>
```

Comment:
```suggestion
            <div className="pull-right">Characters used: {`${notesWordCount}/250`}</div>
```
for consistent spacing.  It's just my liking though.